test/Microsoft.Sbom.Api.Tests/Workflows/ManifestGenerationWorkflowTests.cs

```diff
@@ -272,15 +272,15 @@ public async Task ManifestGenerationWorkflowTests_Succeeds(bool deleteExistingMa
             new FileListEnumerator(fileSystemMock.Object, mockLogger.Object));
 
         var cgPackagesProvider = new CGScannedPackagesProvider(
-            configurationMock.Object,
-            new ChannelUtils(),
-            mockLogger.Object,
-            sbomConfigs,
-            new PackageInfoJsonWriter(
-                manifestGeneratorProvider,
-                mockLogger.Object),
-            packageInfoConverterMock.Object,
-            new PackagesWalker(mockLogger.Object, mockDetector.Object, configurationMock.Object, sbomConfigs));
+                configurationMock.Object,
```

Comment:
I see misaligned indentation in multiple places. Was this changed automatically or you followed any IDE suggestion? Makes me wonder maybe I setup wrong tab length in .editorconfig